cmd/mount_test.go

```diff
@@ -405,8 +405,8 @@ func (suite *mountTestSuite) TestFuseOptions() {
 		{opt: "suid", ignore: true},
 		{opt: "nosuid", ignore: true},
 		{opt: "delay_connect", ignore: true},
-		{opt: "uid=1000", ignore: true},
-		{opt: "gid=1000", ignore: true},
+		{opt: "uid=1000", ignore: false},
```

Comment:
Should we move these params down to belong with non ignored params?

Reply:
done